src/store/modules/order/actions.ts

```diff
@@ -0,0 +1,61 @@
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import OrderState from './orderState'
+import * as types from './mutation-types'
+import { OrderService } from '@/services/OrderService'
+import { hasError, showToast } from '@/utils'
+import { translate } from '@/i18n'
+
+const actions: ActionTree<OrderState, RootState> = {
+  async getDraftOrder ({ commit }, params) {
+    const payload = {
+      draftOrderId: params.id,
+      shopifyConfigId: params.configId
+    }
+    try {
+      const resp = await OrderService.getDraftOrder(payload);
+      if (resp.status === 200 && !hasError(resp) && resp.data.response.draft_order) {
+        const productId = resp.data.response.draft_order.line_items.map((item: any) => item.sku );
+        const store = this.getters['shop/getStore'];
+        if(store){
+          this.dispatch('stock/checkInventory', {productId, facilityId: store.storeCode});
```

Comment:
I think you should create a separate action where you pass the productIds and it will get inventory for all the available facilities